sklearn/cluster/dbscan_.py

```diff
@@ -137,6 +138,9 @@ def dbscan(X, eps=0.5, min_samples=5, metric='minkowski',
     # label_num is the label given to the new cluster
     label_num = 0
 
+    # Candidates
+    ctmp = np.zeros(X.shape[0], dtype=np.int8)
```

Comment:
Re-reading the code I think that `candidates_mask` would be a more descriptive name.